storage/sql/migrate.go

```diff
@@ -98,6 +98,7 @@ var migrations = []migration{
 				claims_email text not null,
 				claims_email_verified boolean not null,
 				claims_groups bytea not null, -- JSON array of strings
+				claims_locale text not null,
```

Comment:
:x: This won't work, I'm afraid. _Migrations_ are always run, even if the tables already exist because the service had been started before. So, to add a column, you'll need to `ALTER TABLE ... ADD COLUMN` in its own migration.